index.html

```diff
@@ -2,13 +2,46 @@
 
 <html>
   <head>
-    <title>vaadin-progress-bar</title>
+    <title>&lt;vaadin-progress-bar&gt;</title>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <script src="../webcomponentsjs/webcomponents-lite.js"></script>
-    <link rel="import" href="../iron-component-page/iron-component-page.html">
+    <style>
+      body {
+        margin: 0;
+      }
+
+      api-viewer {
+        margin: 1rem auto 0;
+      }
+
+      @media(max-width: 799px) {
+        api-viewer {
+          margin: 0;
+        }
+      }
+    </style>
   </head>
   <body>
-    <iron-component-page src="vaadin-progress-bar.html"></iron-component-page>
+    <api-viewer src="custom-elements.json" exclude-knobs="autofocus,items,tabIndex">
+      <template data-element="vaadin-progress-bar" data-target="knob" data-attr="dir" data-type="select">
+        <select>
+          <option value=""></option>
+          <option value="ltr"></option>
+          <option value="rtl"></option>
+        </select>
+      </template>
+      <template data-element="vaadin-progress-bar" data-target="knob" data-attr="theme" data-type="select">
+        <select>
+          <option value="contrast"></option>
```

Comment:
Please add empty option: `<option value=""></option>` to allow deselect `theme` value.

Reply:
done